pipeline/04-interpret.R

```diff
@@ -227,17 +227,25 @@ if (comp_enable) {
   # comp indexes by 1
   comps[[1]] <- comps[[1]] + 1
 
-  # Translate comp indexes to PINs
+  # Translate comp indexes to PINs and doc nos
   comps[[1]] <- comps[[1]] %>%
-    mutate(across(everything(), \(idx_row) {
-      training_data[idx_row, ]$meta_pin
-    })) %>%
+    mutate(
+      across(
+        starts_with("comp_idx_"),
+        ~ training_data[., ]$meta_pin,
+        .names = "comp_pin_{str_remove(col, 'comp_idx_')}"
+      ),
+      across(
+        starts_with("comp_idx_"),
+        ~ training_data[., ]$meta_sale_document_num,
+        .names = "comp_doc_no_{str_remove(col, 'comp_idx_')}")
```

Comment:
There's a variety of different ways we refer to this type of data -- alternately, `doc_no` or `instruno` or `document_num`. What's the best choice for this particular use-case?